### PR TITLE
Fix #82 Remove external references

### LINKED ISSUE #82: Remove functionality involving external systems
- [ ] URLS pointing to outside resources (CSS, JS, fonts, etc)
- [ ] All Social Media Integration (Facebook, Twitter, etc)
- [ ] Rest interface /server/restapi/restapi.coffee
- [ ] integration packages (e.g. irc)

### PR DESCRIPTION
**Pull request by rwakida**

### CODE REVIEW

**Comment by berkich** [comment]
:+1: Looks good

File: client/views/main.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	<meta name="rating" content="general" />
 	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" />
 	<meta name="apple-mobile-web-app-title" content="Rocket.Chat">
-	<meta name="application-name" content="Rocket.Chat">
+	<meta name="application-name" content="Chat Locker">
 	<meta name="msapplication-TileColor" content="#04436a">
 	<meta name="msapplication-TileImage" content="/images/logo/mstile-144x144.png?v=3">
 	<meta name="msapplication-config" content="/images/logo/browserconfig.xml?v=3">
```
Comment (bbrockman) [inline client/views/main.html:15]: Do we need line 17

```
<link rel="canonical" href="https://demo.rocket.chat/" />
```

**Comment by bbrockman** [comment]
Do we need the following files:
 .meteor/cordova-plugins file?
client/lib/constallatoins.js (wasn't this removed from the RC login?
build-android.sh
build-ios.sh
pm2.production.json
/server/methods/getAvatarSuggestion.coffee
/server/startup/migrations/v15.coffee (or at least the method starting on line 62)

Also, do we need all the rocketchat packages? Hubot, external, webrtc, webrtc-ib (maybe others)?

**Comment by rwakida** [comment]
Please retest.

**Comment by berkich** [comment]
Should we make modifications to the names and references in app.json

**Comment by berkich** [comment]
Successfully tested new installation without network access and browsed the packages/source code for any obvious unneeded code :+1:

**Pull request merged**